_i18n/en.yml

```diff
@@ -330,9 +330,9 @@ faq:
   acontribute4: "The Outreach workgroup wrote a useful article to help newcomers:"
   q1: How does Monero have value?
   a1: Monero has value because people are willing to buy it. If no one is willing to buy Monero, then it will not have any value. Monero’s price increases if demand exceeds supply, and it decreases if supply exceeds demand.
-  q2: How can I get Monero?
-  a2: You can buy Monero from an exchange or from an individual. Alternatively, you can try mining Monero to get coins from the @block reward.
-  a2exchanges: "In past, you needed Bitcoin to buy Monero, but that's not the case anymore. You can directly trade Monero for national currencies (USD, EUR, GBP, etc) or other cryptocurrencies on many exchanges. Some require KYC (proof of identification); others do not, like decentralized exchanges. On this website is available a list of exchanges where it's possible to buy/sell Monero (XMR):"
+  q2xmrfiat: How can I get Monero?
+  a2xmrfiat: You can buy Monero from an exchange or from an individual. Institutional exchanges can have very higher fees than local exchanges, i.e. peer-to-peer transaction sites, but beware of scammers as usual... Alternatively, you can try mining Monero to get coins from the @block reward. Monero is legal is most countries, however, it is illegal or banking banned in the following countries: Algeria, Bangladesh, Bolivia, Cambodia, Canada, China, Colombia, Ecuador, Egypt, Indonesia, Iran, Jordan, Morocco, Nepal, Pakistan, Russia, Saudi Arabia, Taiwan, Vietnam.
```

Comment:
Where did you get this list of countries? and what do you mean with `it is illegal or banking banned`

Either way i don't think would be wise to add a list of countries would be wise. It would need to be constantly maintained or people could be misinformed.

Reply:
The list is removed, it was from [wikipedia](https://en.wikipedia.org/wiki/Legality_of_bitcoin_by_country_or_territory). Regarding Canada, the ban means: 

«Companies dealing in virtual currencies must register with the Financial Transactions and Reports Analysis Centre of Canada (Fintrac), implement compliance programs, keep the required records, report suspicious or terrorist-related transactions, and determine if any of their customers are "politically exposed persons."»